src/engraving/layout/layoutsystem.cpp

```diff
@@ -381,8 +381,9 @@ System* LayoutSystem::collectSystem(const LayoutOptions& options, LayoutContext&
     // proportional to the difference between the current length and the target length.
     // After few iterations, the length will converge to the target length.
     qreal newRest = systemWidth - curSysWidth;
-    if (ctx.curMeasure == 0 && ((curSysWidth / systemWidth) <= score->styleD(Sid::lastSystemFillLimit))) {
-        // We do not stretch last system if curSysWidth is <= lastSystemFillLimit
+    if ((ctx.curMeasure == 0 || system->lastMeasure()->sectionBreak())
```

Comment:
Could use `lm->sectionBreak()` here. And check whether lm exists first...

Reply:
Actually that check does fix the crash: `if ((ctx.curMeasure == 0 || (lm && lm->lastMeasure()->sectionBreak()))`

Reply:
yup, good catch! thanks